client-web/wct.conf.json

```diff
@@ -0,0 +1,9 @@
+{
+  "verbose": true,
+  "persistent": true,
+  "plugins": {
+    "local": {
+      "browsers": ["chrome", "firefox", "ie"]
```

Comment:
Let's replace with `"browsers":["all"]`. It prevents fails on different platforms.